bitmap-plasma/app/build.gradle

```diff
@@ -8,6 +8,11 @@ android {
         applicationId 'com.example.plasma'
         minSdkVersion 8
         targetSdkVersion 23
+        externalNativeBuild {
+            cmake {
+                arguments '-DANDROID_PLATFORM=android-9', '-DANDROID_TOOLCHAIN=clang'
```

Comment:
`ANDROID_PLATFORM=android-9` shouldn't be necessary if you already have `minSdkVersion 8`...